torch/utils/data/dataloader.py

```diff
@@ -93,6 +93,9 @@ class DataLoader(Generic[T_co]):
         num_workers (int, optional): how many subprocesses to use for data
             loading. ``0`` means that the data will be loaded in the main process.
             (default: ``0``)
+        prefetch_factor (int, optional): Number of sample loaded in advance by each worker.
+            ``2`` means there will be ``2 * num_workers`` get prefetched. 
```

Comment:
nit: "means there will be a total of ``2 * num_workers`` samples prefetched across all workers".

Reply:
Thanks! Updated.